HintList.py

```diff
@@ -520,51 +520,51 @@ def tokens_required_by_settings(world):
+    'Dodongos Cavern Upper Business Scrubs':                       ("in #Dodongo's Cavern a pair of scrubs# sell...^", None, 'dual'),
     'Dodongos Cavern MQ Larvae Room':                              ("amid #larvae in Dodongo's Cavern# a chest and a spider hold...^", None, 'dual'),
     'Fire Temple Lower Loop':                                      ("under the #entrance of the Fire Temple# a blocked path leads to...^", None, 'dual'),
     'Fire Temple MQ Lower Loop':                                   ("under the #entrance of the Fire Temple# a blocked path leads to...^", None, 'dual'),
-    'Water Temple River Loop Chests':                              ("#chests past a shadowy fight# in the Water Temple hold...^", None, 'dual'),
-    'Water Temple River Checks':                                   ("past a #temporal stone in the Water Temple#, a spider and a chest hold respectively...^", None, 'dual'),
+    'Water Temple River Loop Chests':                              ("#chests past a shadowy fight# in the Water Temple hold...^", "#chests past Dark Link# in the Water Temple hold...^", 'dual'),
+    'Water Temple River Checks':                                   ("in the #river in the Water Temple# lies...^", None, 'dual'),
     'Water Temple North Basement Checks':                          ("the #northern basement of the Water Temple# contains...^", None, 'dual'),
     'Water Temple MQ North Basement Checks':                       ("the #northern basement of the Water Temple# contains...^", None, 'dual'),
     'Water Temple MQ Lower Checks':                                ("#a chest and a crate in locked basements# in the Water Temple hold...^", None, 'dual'),
-    'Spirit Temple Colossus Hands':                                ("upon the #Colossus's right and left hands# are respectively...^", None, 'dual'),
-    'Spirit Temple Child Lower':                                   ("#in between crawl spaces# in the Spirit Temple chests contain...^", None, 'dual'),
-    'Spirit Temple Child Top':                                     ("on the path to the #western Iron Knuckle of the Spirit Temple# a chest and a spider hold respectively...^", None, 'dual'),
-    'Spirit Temple Adult Lower':                                   ("#past the entrance's silver block in the Spirit Temple# boulders and a melody guard...^", None, 'dual'),
-    'Spirit Temple MQ Child Top':                                  ("on the path to the #western Iron Knuckle of the Spirit Temple# a chest and a spider hold respectively...^", None, 'dual'),
-    'Spirit Temple MQ Symphony Room':                              ("#beyond the symphony room# in the Spirit Temple a chest and a spider hold...^", None, 'dual'),
+    'Spirit Temple Colossus Hands':                                ("upon the #Colossus's hands# right and left hands lie...^", None, 'dual'),
```

Comment:
This needs more work. Right now it says “hands” twice, but removing the first would mean only “Colossus's” is colored. Two possibilities:
```suggestion
    'Spirit Temple Colossus Hands':                                ("upon the right and left #hands of the Colossus# lie...^", None, 'dual'),
```
```suggestion
    'Spirit Temple Colossus Hands':                                ("upon the #Colossus's right and left hands# lie...^", None, 'dual'),
```